modules/core/src/test/java/org/apache/ignite/internal/processors/cache/persistence/snapshot/incremental/AbstractIncrementalSnapshotBlockingTest.java

```diff
@@ -326,7 +326,7 @@ public void awaitSnpBlockedOrFinished(@Nullable IgniteFuture<?> snpFut) {
             if (latch == null)
                 return;
 
-            snpFut.listen((f) -> latch.countDown());
+            snpFut.listen(ignored -> latch.countDown());
```

Comment:
ignored -> () ?

Reply:
It's an IgniteFuture (which is a Public API) that was not changed at this PR.

Reply:
Rolled back :(